anvio/splitter.py

```diff
@@ -803,64 +842,133 @@ def process(self, skip_init=False):
                               yet anvi'o failed you? If that's the case, let us know :( This class will quietly\
                               kill this process without reporting any error since a lack of hit may be the\
                               expected outcome of some weird processes somewhere.")
+            return
 
         self.contigs_db = dbops.ContigsSuperclass(self.args, r=self.run_object)
         self.contigs_db.init_functions()
-        counter = 1
-        for gene_callers_id in self.gene_caller_ids_of_interest:
+
+        """
+        Here we will differentiate between being in default-mode OR flank-mode. If in default-mode, we will iterate through
+        the gene-caller-ids-of-interest and cut out the locus X amount of genes above and below the gene-caller-id
+        based on the --num-genes given by the user. If in flank-mode, we will only export 1 locus based on the flanking
+        gene-caller-ids provided.
+        """
```

Comment:
it is a great idea to add comments to code and thanks for doing it, but for those using comment characters is much better. because technically, when the Python interpreter sees this, it interprets it as text. which is not that critical, but it is just not the best practice :) This is ok under function headers where they serve and interpreted as help for the function.

Reply:
Fixed